package.json

```diff
@@ -65,12 +65,17 @@
                 "command": "ros.startCore",
                 "title": "Start Core",
                 "category": "ROS"
-            },
+            },            
             {
                 "command": "ros.stopCore",
                 "title": "Stop Core",
                 "category": "ROS"
             },
+            {
+                "command": "ros.reStartCore",
```

Comment:
@sofu456 Thanks for sending a pull request. One question here. Is `ros.reStartCore` related to the original terminal environment problem? Or it is a independent enhancement, which might be better to be separated into a different pull request?

Reply:
ok，you are right. but i do not know how to correct this,maybe another pull requeset should be required. could you please pull a little not whole?

Reply:
Thanks for your confirmation. I am merging this fix now.